clang/include/clang/Basic/Attr.td

```diff
@@ -4894,6 +4894,14 @@ def HLSLSV_GroupIndex: HLSLAnnotationAttr {
   let Documentation = [HLSLSV_GroupIndexDocs];
 }
 
+def HLSLVkBinding : InheritableAttr {
+  let Spellings = [CXX11<"vk", "binding">];
+  let Subjects = SubjectList<[HLSLBufferObj, ExternalGlobalVar], ErrorDiag>;
+  let Args = [IntArgument<"Binding">, IntArgument<"Set", 1>];
```

Comment:
The default value is 1 here no? Doc says default to 0

Reply:
I doubled checked. The `1` is not the default value. It is a boolean value that says the argument is optional. The default value is provided when we parse the attribute.

Reply:
Ah good to know, thanks for checking